pages/implementation/start-custom-destination.mdx

```diff
@@ -116,6 +120,20 @@ The callback methods mirror the actions you set up for given event in your track
 
   Example: `analytics.setUserProperties(userId, userProperties)`
 
+- `setGroupProperties`: You can add various group properties to events in the Avo dashboard. When an Avo function with attached groups and group properties is called, this callback is triggered.
```

Comment:
I'd feel more comfortable with "You can update various group properties along with events" or something along those lines. It's a subtle difference but I think it would help the user to visualize better both the callbacks happening and the intentions.